requirements/django-dev.pip

```diff
@@ -1,2 +1,2 @@
 -r common.pip
-http://github.com/django/django/tarball/master
+https://github.com/django/django/archive/1.11a1.zip
```

Comment:
Yes, this is intentional.
In tox.ini in section [testenv:docs] are installed requirements from /requirements/common.pip and now is not working for master because, currently master is a Django 2.0.
Then section docs must be tested with python 2.7 and Django < 2.0.

I think that requirements/django-dev.pip and tests for documentation must be changed in future.
Maybe you have any suggestions how to change it?